agent/php_user_instrument.c

```diff
@@ -699,7 +643,6 @@ void nr_php_op_array_set_wraprec(zend_op_array* op_array,
   op_array->reserved[NR_PHP_PROCESS_GLOBALS(zend_offset)] = (void*)index;
 }
 
-#if ZEND_MODULE_API_NO < ZEND_7_4_X_API_NO
```

Comment:
Is there a reason not to exclude it?

Reply:
`nr_php_op_array_get_wraprec` and `nr_php_op_array_set_wraprec` are not built for PHP 7.4+. There's an `#if ZEND_MODULE_API_NO >= ZEND_7_4_X_API_NO` on line [579](https://github.com/misiekn/newrelic-php-agent/blob/refactor/wraprec-lookup/agent/php_user_instrument.c#L579) with an ending `#else` on line [590](https://github.com/misiekn/newrelic-php-agent/blob/refactor/wraprec-lookup/agent/php_user_instrument.c#L590).

Reply:
Thanks for the links, I see it now.